console/console-init/ui/src/components/common/NetworkStatusAlert.tsx

```diff
@@ -34,7 +40,7 @@ const NetworkStatusAlert: React.FunctionComponent = () => {
           title="Disconnected from server"
           action={<AlertActionCloseButton onClose={onClose} />}
         >
-          <span>We're having trouble connecting to our server!</span>
+          <span>{errorMessage}</span>
         </Alert>
         <br />
         <a href="oauth/sign_in" className="pf-c-nav__link">
```

Comment:
In the 403 redirect the user `oauth/sign_in`.  For the other HTTP server errors, forcing them to login again seems too much.   Some server errors (such as a server being restarted) will be transient and the user won't need to login again.   For these, giving a link to `/` is sufficient.

Reply:
Done, I am letting the user redirect to sign in only in case of 403, for rest of the network related errors, the users get the option to get redirected to home.